app/controllers/dishes_controller.rb

```diff
@@ -1,2 +1,14 @@
 class DishesController < ApplicationController
+
+  def create
+    @dish = Dish.create!(dish_params)
+    json_response(@dish, :created)
```

Comment:
I have never really seen this json_response method before, where does it come from?

Typically I would do something like
```
render json: @dish
```

Reply:
scratch that I saw your concern haha!